arrows/core/metadata_map_io_csv.cxx

```diff
@@ -160,6 +226,24 @@ metadata_map_io_csv
 {
 }
 
+// ----------------------------------------------------------------------------
+void
+metadata_map_io_csv
+::set_configuration( vital::config_block_sptr config )
+{
+  d_->write_remaining_columns = config->get_value< bool >(
+    "write_remaining_columns" );
+  d_->write_enum_names = config->get_value< bool >( "write_enum_names" );
+
+  auto const names_string = config->get_value< std::string >( "column_names" );
+  auto const untrimed_column_names = split( names_string, ',' );
+
+  for( auto const& name : untrimed_column_names )
+  {
+    d_->column_names.push_back( trim( name ) );
+  }
+}
```

Comment:
Hmm, this is just begging to be written:
```suggestion
  auto const untrimed_column_names = split( names_string, ',' );
  auto trimmed_column_names = untrimmed_column_names | kvr::transform( trim );
  d_->column_names = { trimmed_column_names.begin(),
                       trimmed_column_names.end() }
```
(...and possibly with a `.reserve` in there if you fancy.)

Pity we don't have ranges yet and can't just write:
```c++
  d_->column_names = split( names_string, ',' ) | kvr::transform( trim );
```

Reply:
L240 returns `kwiver::vital::range::transform_view<std::__cxx11::string &(*)(std::__cxx11::string &), kwiver::vital::config_block_keys_t &>` so the next one breaks.

Reply:
...really? (Grr, I wonder if that's from containers not accepting heterogeneous iterators for construction, yet... another ranges thing, probably. Eh, `std::copy` should work.)

...or `for( auto const& name : untrimed_column_names | kvr::transform( trim ) )` definitely should work, though at that point, meh...

Maybe it's just not worth it until ranges lets us write it sensibly.